bitnami/postgresql/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: postgresql
 sources:
   - https://github.com/bitnami/bitnami-docker-postgresql
   - https://www.postgresql.org/
-version: 10.5.3
+version: 10.5.4
```

Comment:
As it is adding a non breaking feature, could you bump the minor version?

Reply:
Sure no prob. Pushing now